### DISCUSSION TITLE
Morphologist and bids database

### DISCUSSION THREAD

**Discussion by ci2c** [brainvisa/brainvisa.github.io#161] [Q&A] [answered]
Hello the community, sorry but I'm totally lost ;-( 
I followed the installation process, and that went smooth. (At the exception that from brainvisa I could not see my NAS (network storage) so I copy the data in the same folder as the executable, I presume that come from binding at the singularity level.)
But Now that I saw my files from brainvisa, I'm wondering how to process morphologist on the whole bids database, there is no import bids database neither in "Morphologist" nor in "Data Management". Please, is there a kind of "kick start guide" to do so or a kind soul to help me ? thousands of thanks in advance.

**Comment by denisri** [comment, marked as answer]
Hi,
Sorry I totally forgot to answer !
The NAS problem should just be a matter of mounting the NAS directories in the BrainVisa container: as Singularity is a container, its environment is mostly isolated from the host machine system. But you can add mount points: run the "bv" program without parameters, and it will display a configuration GUI, where you can set them up.
For the Bids data processing, we have not used it much so far because we have databases organized in our way for a long time, but there is actually a little of BIDS support in Brainvisa: in the preferences / databases you can add a database pointing to a raw Bids data directory, and you can specify (in the expert settings) that its ontology is "raw-bids-1.0". This should enable it to see and process raw bids input data. However outputs should be in another brainvisa database (because "raw-bids" doesn't organize processed data), that you should also add in the preferences. The output database may be organized as "morphologist-bids-1.0" (which will organize outputs in a kind of bids derivative directories tree), or the regular "brainvisa-3.2.0".
Denis

**Reply by romainviard** [reply]
Thanks to take the time to reply. I managed a solution that works for me.
Good luck
